wsynphot/base.py

```diff
@@ -79,6 +90,20 @@ def list_filters():
 
     return get_filter_index()
 
+#To check whether both passed strings are float or not
```

Comment:
try it and if it raises `ValueError` just leave that `ValueError` through

Reply:
Actually its purpose is for cleaning the data. It returns boolean for each element in dataframe because we need a series to clean the data by boolean indexing.

It might appear confusing here, it will become clear in new PR I'm creating.